src/ground_main.c

```diff
@@ -332,3 +354,177 @@ const char *sub_80990B8(void)
         return sub_8098FB4();
     }
 }
+
+// https://decomp.me/scratch/1QxJZ
+
+NAKED
+u8 sub_80990EC(struct unkStruct_20398C8 *param_1, s16 param_2)
```

Comment:
Here, I managed to match it
https://decomp.me/scratch/ufV6i

Reply:
Thanks